config/standalone.example.toml

```diff
@@ -152,6 +152,36 @@ auto_flush_interval = "1h"
 # Global write buffer size for all regions.
 global_write_buffer_size = "1GB"
 
+# Mito engine options
+[[region_engine]]
+[region_engine.mito]
```

Comment:
I believe it would be beneficial to reorganize the current configuration setup. It appears to be quite disorganized and could benefit from being split into separate sections, such as "compaction," "manifest," and "cache," among others.

Reply:
It's not blocking, but may be a good first issue.

Reply:
We can consider this after `mito` has more options. Now each section has no more than 3 options.